Dockerfile

```diff
@@ -3,6 +3,7 @@ FROM centos:6
 WORKDIR /opt
 
 ENV JAVA_HOME=/usr/lib/jvm/java-1.7.0-openjdk.x86_64
+ENV PERSEO_FE_URL=172.17.0.1:19090
```

Comment:
This seems to be a kind of hardwiring. We cannot assure that Perseo FE will be listening on that IP/port...